proposals/KEEP-0389-kdoc-streamline-KDoc-ambiguity-references.md

```diff
@@ -1,24 +1,29 @@
 ﻿# Streamline KDoc ambiguity links
 
-* **Type**: KDoc proposal
+* **Type**: Proposal
 * **Author**: Vadim Mishenev
 * **Status**: Submitted
```

Comment:
Can you also please update the status according to https://github.com/Kotlin/KEEP?tab=readme-ov-file#keep-document-lifecycle